### PR TITLE
Update release-notes-0-18-0.md

### PR DESCRIPTION
**Pull request by diana-macrometa**
Add STRM-122 to the defect list.

### CODE REVIEW

**Comment by diana-macrometa** [comment]
@SpringNgMacrometa and @sourabhedake-mm - Please review this PR.

File: docs/release-notes/release-notes-0-18-0.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ The following defects were fixed in this release.
 | GUI-1823  | Fixed dashboard typo in CPU Usage Global Metrics; graph showed value in seconds instead of ms.  |
 |   |   |
 |   |   |
-|   |   |
+| STRM-122  | Streams can be created with invalid stream names, such as "123" or "_#".  |
```
Comment (StoyanTc) [inline docs/release-notes/release-notes-0-18-0.md:184]: @diana-macrometa - `STRM-122` disallows creating streams with invalid names. Please correct the note.

Comment (diana-macrometa) [reply docs/release-notes/release-notes-0-18-0.md:184]: This is the defect that was fixed. It's a list of things that were wrong, now no longer are.

**Review by StoyanTc — APPROVED** [review]

File: docs/release-notes/release-notes-0-18-0.md (hunk 1 above)
Comment (StoyanTc) [reply docs/release-notes/release-notes-0-18-0.md:184]: Got it, approved.

**Pull request merged**